File: 7th-homework/problemSet/mario.py
# TODO
from cs50 import get_int

height = get_int("Height: ")
while 1 > height or height > 8:
    print("This input as invalid")
    height = get_int("Height: ")


for i in range(height):
    print(" " * (height - i - 1), end="")
    print("#" * (i + 1))

File: 2nd-homework/mario.c
#include <cs50.h>
#include <stdio.h>

void draw(int n);

int main(void)
{
    int n;
    do
    {
        n = get_int("Height: ");
    }
    while(n < 1 || n >= 9);

    draw(n);
}

void draw(int n)
{
    for(int i = 1; i <= n; i++)
    {
        for(int k = n - i; k > 0; k--)
        {
            printf(" ");
        }
        for(int k = i; k > 0; k--)
        {
            printf("#");
        }
        printf("  ");
        for(int k = i; k > 0; k--)
        {
            printf("#");
        }
        printf("\n");
    }
}